### PR TITLE
Custom scripts/sandbox2: Added two WDL files (one for Wizard class and one for Warrior class) for custom-scripts demo 2022

### LINKED ISSUE #1636: Adding two WDL files for each class that is used for custom-scripts demonstration 2022
Different WDL files will be loaded into the game depending on the output of the LUA file. There is one WDL specifically for the wizard class and another one for the Warrior Class. Each WDL associated with its respective class have different items and actions.

### PR DESCRIPTION
**Pull request by jackjackkimkim**
This is the pull request which resolves issue #1636:

There are two WDL files and each is associated with a player class. There is one for the wizard class and one for the warrior class. We will work on implementing these two respective WDL files into our 2022 demonstration's Lua script so that choosing a class can load in different versions of Chiventure, with different abilities in regards to item interaction. This issue is closed when a merge request is completed which contains the two WDL files for each class.

### CODE REVIEW

**Review by RamarkoB — APPROVED** [review]
Looks Good! All the Demos are compiling for me, and the it passes all the tests! Good work on expanding the demo to be more dynamic!

File: src/custom-scripts/examples/loadwdl2.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    char string_num;
+    printf("Enter either 1 or 2 (1 for wizard class, 2 for warrior class): ");
+    scanf("%c", &string_num); 
+    dc.c = string_num;
+    togay = obj_add_arg(togay,dc,CHAR_TYPE);
+    char* custom_string = (char*)malloc(100);
+    data_t temp = arg_t_get(togay);
+    custom_string = temp.s;
+    char wizard[] = "Wizard Class", warrior[] = "Warrior Class";
+    if(strcmp(wizard,custom_string)== 0){
```
Comment (ecoble) [inline src/custom-scripts/examples/loadwdl2.c]: Please follow the style guide for adding spaces like this: `if (` and this `) {`

**Review by ecoble — CHANGES_REQUESTED** [review]

File: src/custom-scripts/examples/loadwdl2.c (hunk 1 above)
Comment (RamarkoB) [reply src/custom-scripts/examples/loadwdl2.c]: Fixed!

**Review by ecoble — APPROVED** [review]

**Comment by tatmaca** [comment]
This issue is now closed! Both WDL files are read correctly by the Chiventure game and in both demonstrations the correct items are being loaded in with the proper associated actions. Since everything runs as it should, all tests have passed, and the merge request went through, this pull request is closed!

**Comment by ecoble** [comment]
Issue Score: Excellent

Comments:
Great job on getting this working for your demo, it was really effective!

**Pull request merged**